src/python/pants/backend/terraform/target_types.py

```diff
@@ -18,5 +18,18 @@ class TerraformModule(Target):
     help = """A single Terraform module."""
 
 
+class TerraformModulesSources(TerraformSources):
```

Comment:
FYI: The `help` should probably be rewritten, but you can punt on that. I'm still thinking through whether target generators should have a different `Sources` class than "atomic targets". (Field name would be the same, different Python class.)

Reply:
Yeah not sure where I come out on that byeond the fact that I chose to have a separate sources field for `terraform_modules` than for `terraform_module` because I could change the default glob. They both inherit directly from `TerraformSources` so that dependency inference can look for `TerraformModuleSources` (vs `TerraformModulesSources`,  not a fan of the one letter addition though). So really just practical concerns drove the choice.